dataretrieval/wqp.py

```diff
@@ -57,10 +57,10 @@ def get_results(**kwargs):
     kwargs['zip'] = 'no'
     kwargs['mimeType'] = 'csv'
 
-    response = query(wqp_url('Result'), **kwargs)
+    response = query(wqp_url('Result'), list(kwargs.items()))
```

Comment:
query parameters can have multiple keys with different values.  kwargs is a dictionary, which only can have one value per key.  This is why get_pmcodes was not working, the show parameter has 4 parameters with the keyword show.  Since it was getting placed into a dictionary, only the last value was being used.